### ISSUE TITLE
Add a setting to disable mouse-wheel scrolling

### ISSUE THREAD

**Issue by zadjii-msft** [microsoft/terminal#3990]
This is in lieu of having actual mouse bindings. Some people might actually want the mouse to not scroll, or might want to pass the mouse scrolling through.

Until we have proper mouse bindings, this settings should suffice.

###### note 11/8/2021

This task is a "instead of #1553" task. If we just do #1553, then this isn't relevant. 

###### note 2/22/2022

I'd add this as a profile setting, `experimental.mouseWheelScrolling: bool`. Probably just the same as #11710. Since we're using this as an "escape hatch" issue while we wait for the architecture to do #1553 right, I'd just go ahead and disable all wheel scrolling with this setting - including opacity scrolling and zooming.

**Comment by bloem777** [comment]
My experience is that scrolling in a tmux window not possible at the moment.

**Comment by zadjii-msft** [comment]
@bloem777 That's correct, mouse mode doesn't work quite yet. Please refer to #545, #376

**Comment by andreykaipov** [comment]
I added `"historySize": 0` to my profile to disable scrolling.

Can't scroll if there's nothing to scroll to! 🙃

**Comment by alanlivio** [comment]
It is a very interesting feature.

**Comment by SWARAJSINGH211** [comment]
is it okay a setting to dsable mouse-whell scrollng 
#include <windows.h>

int main() {
  // Create a dialog box with a checkbox to disable mouse-wheel scrolling.
  HWND hWndDialog = CreateDialog(NULL, IDD_DIALOG1, NULL, NULL);

  // Show the dialog box.
  ShowWindow(hWndDialog, SW_SHOW);

  // Get the message loop for the dialog box.
  MSG msg;
  while (GetMessage(&msg, NULL, 0, 0)) {
    TranslateMessage(&msg);
    DispatchMessage(&msg);
  }

  // Get the state of the checkbox.
  BOOL fDisableMouseWheelScrolling = IsDlgButtonChecked(hWndDialog, IDC_DISABLE_MOUSE_WHEEL_SCROLLING);

  // Disable mouse-wheel scrolling if the checkbox is checked.
  if (fDisableMouseWheelScrolling) {
    SetScrollRate(NULL, 0, 0);
  }

  // Destroy the dialog box.
  DestroyWindow(hWndDialog);

  return 0;
}

**Comment by marcelwgn** [comment]
@SWARAJSINGH211 Do I understand your comment correctly that you would like (or already started/tried) working on this issue? 😊
